tests/test_agents.py

```diff
@@ -242,6 +242,9 @@ async def test_get_directory_index_w_manifest(agent_test_settings: Settings) ->
 
         results = await index.query(query="who is Frederick Bates?")
         top_result = next(iter(results[0].docs.values()))
+        
+        # NOTE: This is failing because now the dockey comes from the manifest.
+        # In the doctdetails object we are setting the dockey to the doc_id
         assert top_result.dockey == md5sum(abs_paper_dir / "bates.txt")
```

Comment:
This test is failing, I will explain why.

When we build the manifest we do 

`records = [DocDetails(**r) for r in csv.DictReader(content.splitlines())]`

and when the DocDetails is constructed, we do

data["dockey"] = data["doc_id"]

So now we are getting everything we can from the manifest (that's the purpose of this PR) but then  `dockey = md5sum(path)` never runs in docs.py

That's why, in the test, the assertion `top_result.dockey == md5sum(abs_paper_dir / "bates.txt")` is false